### ISSUE TITLE
Don't require key with v-for on template tags

### ISSUE THREAD

**Issue by hirokiosame** [vuejs/eslint-plugin-vue#43]
* **ESLint Version:** 3.19.0
* **eslint-plugin-vue Version:** 3.1.3
* **Node Version:** 7.9.0

**What did you do? Please include the actual source code causing the issue.**

```vue

<template>
	<div>
		<template v-for="crumb in crumbArr" :key="crumb">
			<custom-el :crumb="crumb" />
		</template>
	</div>
</template>
```

**What did you expect to happen?**
For it to work.

**What actually happened? Please include the actual, raw output from ESLint.**
No ESLint error. Vue error: `<template> cannot be keyed. Place the key on real elements instead.`

**Comment by mysticatea** [comment]
Thank you for the proposal.

You are right, this plugin should catch it.

**Comment by michalsnik** [comment]
Fixed in `v3.4.0` 🚀
